sdk/identity/azure-identity/src/client_secret_credential.cpp

```diff
@@ -9,6 +9,40 @@
 #include <chrono>
 #include <sstream>
 
+namespace {
+// Assumes !scopes.empty()
```

Comment:
Since this assumes `!scopes.empty()`, should there be an early return?

Reply:
I'd rather not - or otherwise, we need to return an empty string, so the external check for `!scopes.empty()` becomes `!scopesString.empty()`, and that's extra empty work the program will do.